### PR TITLE
grabbed human position fix

### PR DESCRIPTION
**Pull request by Picadi0**
When a human is grabbed by a titan, human position wasn't being updated locally, it was being updated remotely, which caused it to look like this
before
So Here is the fixed version.
after

### CODE REVIEW

File: Assets/Scripts/Characters/Human/Human.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@
 using UnityEngine;
 using UnityEngine.UI;
 using UnityEngine.UIElements;
+using UnityEngine.XR;
```
Comment (AutumnThyme) [inline Assets/Scripts/Characters/Human/Human.cs:25]: Is this being used?

File: Assets/Scripts/Characters/Human/Human.cs [hunk 2]
```diff
@@ -444,7 +445,21 @@ public void Idle()
...
+        [PunRPC]
+        public void setMyGrabber(int grabberViewID, PhotonMessageInfo info)
+        {
+            if (info.Sender != Cache.PhotonView.Owner)
+                return;
+            BaseTitan grabber = PhotonView.Find(grabberViewID).GetComponent<BaseTitan>();
+            if (grabber != null)
+            {
+                Grabber = grabber;
+                if (grabber.GetCurrentAnimation().Contains("grab") && grabber.GetCurrentAnimation().Contains(".l"))
```
Comment (AutumnThyme) [inline Assets/Scripts/Characters/Human/Human.cs:457]: Not crazy about this pattern for checking grabs/direction, maybe create a function for telling what the type of move is and the direction, multiple Contains calls are a bit messy as well but its a short string so as long as it isn't called a lot its fine.

**Review by AutumnThyme — CHANGES_REQUESTED** [review]
Remove XR import if not used and create a function for categorizing grabs instead of copying code.
Something like GetGrabHand(animation)

File: Assets/Scripts/Characters/Human/Human.cs (hunk 1 above)
Comment (Picadi0) [reply Assets/Scripts/Characters/Human/Human.cs]: No it's not used and removed all unused libraries

File: Assets/Scripts/Characters/Human/Human.cs (hunk 2 above)
Comment (Picadi0) [reply Assets/Scripts/Characters/Human/Human.cs]: LOL yea it's looks funny but was working anyway changed it as you wanted

**Comment by Picadi0** [comment]
@AutumnThyme Done

**Review by AutumnThyme — APPROVED** [review]

**Review by AutumnThyme — APPROVED** [review]

**Pull request merged**